sklearn/model_selection/tests/test_search.py

```diff
@@ -342,15 +343,27 @@ def test_return_train_score_warn():
     y = np.array([0] * 5 + [1] * 5)
     grid = {'C': [1, 2]}
 
-    estimators = [GridSearchCV(LinearSVC(random_state=0), grid, iid=False),
-                  RandomizedSearchCV(LinearSVC(random_state=0), grid,
-                                     n_iter=2, iid=False)]
+    estimators = [
```

Comment:
As mentioned somewhere else, try to avoid changing unrelated things, especially if it is only cosmetic. Can you revert this change?